README.md

```diff
@@ -10,4 +10,41 @@ This is the HMPPS Staff lookup service.
 
 This API has been created to allow consumers to search for information relating to HMPPS staff.
 
-Currently there is only one endpoint, which searches for users with e-mails that begin with the given characters. 
\ No newline at end of file
+Currently there is only one endpoint, which searches for users with e-mails that begin with the given characters. 
+
+### Support
```

Comment:
new runbook page? feels like this should be in confluence

Reply:
Yes, but it's too immature to bother right now. I'll raise a ticket to do that when we actually have something to support.

If we do it now we'll only forget

Reply:
Created ticket.. https://dsdmoj.atlassian.net/browse/WFP-1788